### DISCUSSION TITLE
Filter: Add option to show issues that have been edited or edited+locked within Komga

### DISCUSSION THREAD

**Discussion by TBhimdi** [gotson/komga#1659] [Ideas]
I like to keep file metadata in sync with Comic Vine, but very rarely have to update the metadata in Komga instead (e.g., Oneshot names, dates, etc.). I have done that for several files and locked that field, but now I don't know how to find them to unlock those fields to go back to using the cbz metadata.

An option to display or list all the files with Komga-only metadata would be very helpful.

**Comment by gotson** [comment]
There are a lot of metadata fields, how would you see that filter working exactly?

**Reply by TBhimdi** [reply]
"Issues with any locked fields"
"Issues with any fields that don't match ComicInfo.xml"

The filter could also show which field exactly to filter on, but that could get messy in the UI. Once the issues are listed, manually going through each file should be a lot quicker than it is now.

**Reply by gotson** [reply]
> "Issues with any fields that don't match ComicInfo.xml"

this is not possible. Komga does not keep track of the what's in ComicInfo.xml.

> "Issues with any locked fields"

Possible, i'm not sure there's a lot of value in there though.

**Reply by TBhimdi** [reply]
Is there any way to know which issues have had their metadata edited within Komga other than remembering or keeping it track of it manually elsewhere? Honest question, because I would want to revert some field locks in the future.

**Reply by gotson** [reply]
> Is there any way to know which issues have had their metadata edited within Komga other than remembering or keeping it track of it manually elsewhere? Honest question, because I would want to revert some field locks in the future.

no. Komga loads data from all the eligible sources, and updates the metadata if it's not locked. There's no keeping track of the metadata provenance.
